Learn-JSON-fetch-promises-async-await-by-building-an-fCC-forum-leaderboard/index.js

```diff
@@ -0,0 +1,131 @@
+const postsContainer = document.getElementById("posts-container");
+const FORUM_TOPIC_URL = "https://forum.freecodecamp.org/t/";
+const FORUM_CATEGORY_URL = "https://forum.freecodecamp.org/c/";
+const AVATAR_URL = "https://sea1.discourse-cdn.com/freecodecamp";
+
+// this might be the first time campers are working with IFFEs
+(async () => {
+  const res = await fetch("https://forum-proxy.freecodecamp.rocks/latest");
+  const data = await res.json();
+  showLatestPosts(data);
+  // do we want to leave this here so campers can see the output?
+  console.log(data);
+})();
+
+// do we want campers to build this out or just supply it for them?
+const categories = {
+  299: { category: "Career Advice", className: "career" },
+  409: { category: "Project Feedback", className: "feedback" },
+  417: { category: "freeCodeCamp Support", className: "support" },
+  421: { category: "JavaScript", className: "javascript" },
+  423: { category: "HTML - CSS", className: "html-css" },
+  424: { category: "Python", className: "python" },
+  432: { category: "You Can Do This!", className: "motivation" },
+  560: { category: "Backend Development", className: "backend" },
+};
+
+const forumCategories = (id) => {
+  return categories.hasOwnProperty(id)
+    ? `<a class="category ${categories[id]["className"]}"  target="_blank" href=${FORUM_CATEGORY_URL}${categories[id]["className"]}/${id}>${categories[id]["category"]}</a>`
+    : `<a class="category general" target="_blank" href=${FORUM_CATEGORY_URL}general/1>General</a>`;
```

Comment:
```suggestion
    ? `<a class="category ${categories[id]["className"]}" target="_blank" href="${FORUM_CATEGORY_URL}${categories[id]["className"]}/${id}">${categories[id]["category"]}</a>`
    : `<a class="category general" target="_blank" href="${FORUM_CATEGORY_URL}general/1">General</a>`;
```
We want to reinforce that attributes are wrapped in quotes